osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -187,41 +193,39 @@ private double computeSpeedValue()
 
         private double computeAccuracyValue()
         {
-            if (mods.Any(h => h is OsuModRelax))
```

Comment:
Was this intentional?

Reply:
Happened due to master merge, should be fixed.